.github/workflows/deploy.yml

```
@@ -7,7 +7,7 @@ jobs:
     if: github.repository == 'hms-dbmi/gehlenborglab-website'
     # Available versions:
     # https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on
-    runs-on: ubuntu-20.04
```

I'm thinking we could maybe just pin latest and avoid running into this issue again in the future. I guess that runs the risk of a future version not being compatible... but I've not run into that with Ruby/Node projects.

```suggestion
ubuntu-latest
```